### PR TITLE
Docs: Add Initial Results

### PR DESCRIPTION
**Pull request by TobeyQin**

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,20 @@
 
 This repo is used to collect benchmarking results for [SuperBench](https://github.com/microsoft/superbenchmark) project.
 
-You can follow [Contribute Benchmark Results](https://microsoft.github.io/superbenchmark/docs/developer-guides/contributing#contribute-benchmark-results) to upload your results run by SuperBench.
+## How to Get and Upload Results
+
+You can get three files automatically after running SuperBench successfully.
+- **config file**: This is a `YAML` file to configurate parameters when run benchmarks. 
+  
+  SuperBench provides a set of config files for different workloads, you can adjust the parameters according your needs.
```
Comment (yukirora) [inline README.md]: according to

File: README.md [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
+You can get three files automatically after running SuperBench successfully.
+- **config file**: This is a `YAML` file to configurate parameters when run benchmarks. 
+  
+  SuperBench provides a set of config files for different workloads, you can adjust the parameters according your needs. 
+
+- **results summary file**: This file contains the results run by SuperBench. 
+  
+  The default summary results path for each node is `outputs/nodes/$SPECIFIED-NODE-NAME/results-summary.json`. You can also specified the outputs folder by add `--output-dir` paramerter when run SuperBench.
+
+- **system information file**: This is system configurations file and generated by run `system_info.py`.
```
Comment (yukirora) [inline README.md]: This is system configurations file and generated by run `'sb node info'`,  and add the restrict that it must be run under root

File: v0.2.1/Azure-NDv4-A100/azure_ndv4.yaml [hunk 3]
```diff
@@ -0,0 +1,150 @@
+# SuperBench Config
+version: v0.2
+superbench:
+  enable: ['nccl-bw']
```
Comment (yukirora) [inline v0.2.1/Azure-NDv4-A100/azure_ndv4.yaml]: should be null to run all the benchmarks

**Review by cp5555 — APPROVED** [review]

**Comment by yukirora** [comment]
the result json file for a100 and v100 are not right

**Comment by yukirora** [comment]
> the result json file for a100 and v100 are not right

have re-run and upload the new ones

**Review by yukirora — APPROVED** [review]

**Pull request merged**